File: replication/src/test/com/db4o/test/replication/CollectionHolder.java
/* Copyright (C) 2005   db4objects Inc.   http://www.db4o.com */

package com.db4o.test.replication;

import java.util.HashMap;
import java.util.HashSet;
import java.util.LinkedList;
import java.util.List;
import java.util.Map;
import java.util.Set;

public class CollectionHolder {
	public String _name;
	public Map _map = new HashMap();
	public List _list = new LinkedList();
	public Set _set = new HashSet();
	public CollectionHolder _h2;

	public CollectionHolder() {
	}

	public CollectionHolder(String name) {
		_name = name;
	}

	public String toString() {
		return _name + ", hashcode = " + hashCode();
	}

}

File: drs/drs.net/drs.tests/src/com/db4o/drs/test/CollectionHolder.cs
using System.Collections;

namespace com.db4o.drs.test
{
    public class CollectionHolder
    {
        public string name;

        public System.Collections.IDictionary ht = new System.Collections.Hashtable();

        public ArrayList list = new ArrayList();

        public IDictionary set = new Hashtable();

        public CollectionHolder()
        {
        }

        public CollectionHolder(string name)
        {
            this.name = name;
        }

        public override string ToString()
        {
            return name + ", hashcode = " + GetHashCode();
        }
    }
}
